packages/dds/tree/prettier.config.cjs

```diff
@@ -0,0 +1,38 @@
+// Shared prettier configuration for use in across the fluid-framework repository.
```

Comment:
This is not the shared config. This file is in the tree directory. Did you just copy the shared config file here? As documented on the next line "Individual packages may extend" the shared config file. If we don't need any changes from the defaults we could just use the shared config as is though.

Reply:
I asked Josh, and he said tools/api-markdown-documenter has a good example of how to do this properly

Reply:
#11982 is fixing our other configs to follow this pattern.

Reply:
I have updated the config file to just use the shared config (similarly to #11982 ), since we didn't need any additional changes. Will we still need to use the tools/api-markdown-documenter if this is the case?